src/hal/drivers/hal_gm.c

```diff
@@ -1840,7 +1840,7 @@ RS485(void *arg, long period)
 	  {
 	    if((device-> RS485_mgr.ID[2*i]) != ((temp_u32 >> 8) & 0xff)) 
             {
-              //RS485 module falled off, error
+              //RS485 module failed off, error
```

Comment:
probably "fell off" is the correct version

Reply:
Or maybe just `//RS485 module failed, error` ?
"Fall off" does not make much sense in this context IMHO.